RedHatInsights/ocp-advisor-frontend
# feat(OCPADVISOR-144): workloads table filtering & sorting in url params

[OCPADVISOR-144](https://issues.redhat.com/browse/OCPADVISOR-144)

Filtering and sorting is now tracked in the url params. Persists between refreshes.

How to test:
1. Apply some filters, sorting and pagination and verify it is correct in the url.
2. Check if it persists between refreshes.
